### PR TITLE
Fix Dutch translation: submit/apply

### PR DESCRIPTION
**Pull request by NielsCharlier**

### CODE REVIEW

File: src/web/core/src/main/resources/GeoServerApplication_nl.properties [hunk 1]
```diff
@@ -789,7 +789,8 @@ showingAllRecords = Resultaat {0} t/m {1} (totaal\: {2})
 
 stores = Bronnen
 
-submit = Toepassen
+submit = Verzenden
```
Comment (mprins) [inline src/web/core/src/main/resources/GeoServerApplication_nl.properties]: I would rather choose "Opslaan" instead of "Verzenden"

Comment (NielsCharlier) [reply src/web/core/src/main/resources/GeoServerApplication_nl.properties]: Well, it does seem to be used that way although that is not the conventional meaning of submit on the internet (which is normally "verzenden"). There is already "Save" which is translated as "Opslaan" but I'm not really sure why some pages in geoserver use "submit" instead.

Comment (NielsCharlier) [reply src/web/core/src/main/resources/GeoServerApplication_nl.properties]: To give some background, currently the page to create/edit styles has a button "Apply" and a button "Toepassen" and this is obviously very confusing.

Comment (mprins) [reply src/web/core/src/main/resources/GeoServerApplication_nl.properties]: what about "Indienen" ?

Comment (mprins) [reply src/web/core/src/main/resources/GeoServerApplication_nl.properties]: > To give some background, currently the page to create/edit styles has a button "Apply" and a button "Toepassen" and this is obviously very confusing.

yeah, I always hit the wrong button too :confused:

Comment (NielsCharlier) [reply src/web/core/src/main/resources/GeoServerApplication_nl.properties]: Yeah OK, that is fine too. It sounds a bit like giving your homework to the teacher, but it is actually the most accurate translation :)

**Comment by NielsCharlier** [comment]
New version with "indienen"

**Pull request merged**